053-SentinelAutomatedResponse/Coach/Solution-01.md

```diff
@@ -0,0 +1,30 @@
+# Challenge 01 - Architecture, Agents, Data Connectors and Workbooks - Coach's Guide
+
+**[Home](./README.md)** - [Next Challenge>](./Solution-02.md)
+
+## Introduction
+In this Coach's guide, you fill find guidance, code, and examples that will help you guide the teams over the course of the WTH. In the spirit of continuous improvement, update this repository with any suggestions, altertnatives, or additional challenges.
```

Comment:
Consider adding a link to our WTH Contribution Guide here which has the instructions on HOW someone can update the repository with updates!  https://aka.ms/wthcontribute

Reply:
Thanks for the contribution guide link!

The link to your PPT presentation(s) is broken though.  Need to fix that.